packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -96,6 +97,11 @@ class ExpandIcon extends StatefulWidget {
   /// and for [dark theme](https://material.io/design/color/dark-theme.html#ui-application)
   final Color? expandedColor;
 
+  /// An optional [IconData] for the [IconButton].
+  ///
+  /// If not provided, the [IconButton] will default to [Icons.expand_more].
```

Comment:
Might be better with a more concise description that matches the other properties.

```suggestion
  /// The icon used by the button.
  ///
  /// If null, [Icons.expand_more] will be used.
```